images/driver-base/Dockerfile

```diff
@@ -12,15 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM photon:2.0
+FROM photon:3.0
 LABEL "maintainer" "Travis Rhoden <trhoden@vmware.com"
 
 RUN tdnf -y remove toybox
-
+RUN tdnf -y upgrade
```

Comment:
Good catch here. Do you know what the diff would be between `upgrade` and `distro-sync` ?

Reply:
both seems doing the same as per following output.

**tdnf upgrade**

```
% docker run -it photon:3.0 sh
sh-4.4# bash
root [ / ]# tdnf upgrade
Refreshing metadata for: 'VMware Photon Extras 3.0 (x86_64)'
Refreshing metadata for: 'VMware Photon Linux 3.0 (x86_64)'
Refreshing metadata for: 'VMware Photon Linux 3.0 (x86_64) Updates'
photon-updates                           10492    100%
Upgrading:
tdnf-cli-libs                                           x86_64                      2.1.0-2.ph3                           photon-updates                    63.34k 64856
tdnf                                                    x86_64                      2.1.0-2.ph3                           photon-updates                  245.82k 251720
libsolv                                                 x86_64                      0.6.35-3.ph3                          photon-updates                   1.07M 1124860

Total installed size:   1.37M 1441436
Is this ok [y/N]:y

Downloading:
tdnf-cli-libs                            30343    100%
tdnf                                     91795    100%
libsolv                                 352629    100%
Testing transaction
Running transaction
Installing/Updating: libsolv-0.6.35-3.ph3.x86_64
Installing/Updating: tdnf-cli-libs-2.1.0-2.ph3.x86_64
Installing/Updating: tdnf-2.1.0-2.ph3.x86_64
Removing: tdnf-2.0.0-11.ph3.x86_64
Removing: tdnf-cli-libs-2.0.0-11.ph3.x86_64
Removing: libsolv-0.6.35-2.ph3.x86_64

Complete!
root [ / ]# 
```

**tdnf distro-sync**

```
% docker run -it photon:3.0 sh                                                                                                                                                                                         
sh-4.4# bash
root [ / ]# tdnf distro-sync
Refreshing metadata for: 'VMware Photon Extras 3.0 (x86_64)'
Refreshing metadata for: 'VMware Photon Linux 3.0 (x86_64)'
Refreshing metadata for: 'VMware Photon Linux 3.0 (x86_64) Updates'
photon-updates                           10492    100%
Upgrading:
tdnf-cli-libs                                           x86_64                      2.1.0-2.ph3                           photon-updates                    63.34k 64856
tdnf                                                    x86_64                      2.1.0-2.ph3                           photon-updates                  245.82k 251720
libsolv                                                 x86_64                      0.6.35-3.ph3                          photon-updates                   1.07M 1124860

Total installed size:   1.37M 1441436
Is this ok [y/N]:y

Downloading:
tdnf-cli-libs                            30343    100%
tdnf                                     91795    100%
libsolv                                 352629    100%
Testing transaction
Running transaction
Installing/Updating: libsolv-0.6.35-3.ph3.x86_64
Installing/Updating: tdnf-cli-libs-2.1.0-2.ph3.x86_64
Installing/Updating: tdnf-2.1.0-2.ph3.x86_64
Removing: tdnf-2.0.0-11.ph3.x86_64
Removing: tdnf-cli-libs-2.0.0-11.ph3.x86_64
Removing: libsolv-0.6.35-2.ph3.x86_64

Complete!
root [ / ]# 
```